#
#

Header header
float32 velocity        # Speed of the robot in m/s
float32 ackermannAngle  # Ackermann angle in rad, range from -0.61 to 0.61.
